test/e2e/azure_test.go

```diff
@@ -141,7 +142,7 @@ var _ = Describe("Workload cluster creation", func() {
 
 	if os.Getenv("LOCAL_ONLY") != "true" {
 		Context("Creating a private cluster", func() {
-			It("Creates a public management cluster in the same vnet", func() {
+			It("Creates a public management cluster a custom vnet", func() {
```

Comment:
nit: "in a custom vnet"

If this test run passes lgtm and merge :)

Reply:
fixed!